index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="./style.css">
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css">
+
+    <style>
+        /* Stackoverflow preview fix, please ignore */
+        
+        .navbar-nav {
+            flex-direction: row;
+        }
+        
+        .nav-link {
+            padding-right: .5rem !important;
+            padding-left: .5rem !important;
+        }
+    </style>
```

Comment:
> * You have done a very great job of researching the internet to fix issues, how I think it will be much better if you can kindly remove the StackOverflow comment and make the code more personal.
> * Also please move this CSS code to the style.css file. Separating CSS  from HTML keeps the code clean and easy to maintain.

- [ ] [OPTIONAL] This is exciting progress after the first review🚶‍♂️ 👏  . You quoted the code and also informed the student about the reason behind requesting changes ♻️ .  At times to make the student feel more comfortable and friendly with your reply, you may consider using [github emojis](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md)